src/classes/frDonor.cls

```diff
@@ -39,7 +39,7 @@
 */
 
 public class frDonor extends frModel implements frSyncable{
-    public static final String TYPE = 'Donor';
+    public static final String TYPE = frUtil.hasNPCobjects() ? 'Person Account' : 'Donor';
```

Comment:
Note that previously this was not `Contact`, so there shouldn't be a reason to change the Type away from `Donor`, as this is still a Donor type, just using a different underlying SObject

Reply:
Removed!